alpaca_trade_api/rest.py

```diff
@@ -884,6 +898,43 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
                              api_version='v1beta1')
         return self.response_wrapper(resp, SnapshotV2)
 
+    def get_news_iter(self,
+                      symbol: Union[str, List[str]],
```

Comment:
`symbol` should be Optional here and default to `None` to be consistent with the REST API.

Reply:
Totally missed this. Thanks for catching